blocks/library/verse/index.js

```diff
@@ -58,6 +58,7 @@ registerBlockType( 'core/verse', {
 				onFocus={ setFocus }
 				placeholder={ __( 'Write…' ) }
 				className={ className }
+				formattingControls={ [ 'bold', 'italic', 'strikethrough' ] }
```

Comment:
It may also work as an example of how to customize editable. In any case, we can revisit and iterate.